src/Request/RequestHandler.php

```diff
@@ -157,6 +168,11 @@ public function __construct(array $config)
             $this->privateKey = $config['private_key'];
         }
 
+        $this->tokenVerified = false;
+        if (isset($config['token_verified'])) {
```

Comment:
What purpose does this config variable have?

When would this normally come into play?